app/ide-desktop/lib/content/esbuild-config.ts

```diff
@@ -122,6 +123,28 @@ export function bundlerOptions(args: Arguments): esbuild.BuildOptions {
             // We do not control naming of third-party options.
             esbuildPluginAlias({ ensogl_app: ensoglAppPath }),
             esbuildPluginTime(),
+            // This must run before the copy plugin so that the generated `tailwind.css` is used.
+            {
+                name: 'enso-generate-tailwind',
+                setup: build => {
+                    build.onStart(() => {
+                        const source = path.resolve(THIS_PATH, 'src', 'tailwind.css')
```

Comment:
duplicated code line. Exactly the same logic was added above. Please refactor it to a common constant somewhere close to the top of the file.